tacs/mphys/postcoupling.py

```diff
@@ -130,3 +130,6 @@ def setup(self):
                 con_group.add_subsystem(
                     constraint.name, con_comp, promotes_inputs=promotes_inputs
                 )
+
+    def writeSolution(self):
```

Comment:
Can we change this to `write_solution` to match with the mphys/openmdao convention

Reply:
Done